### PR TITLE
[TACHYON-746] Improve Javadoc in RemoteBlockReader

### PR DESCRIPTION
**Pull request by stormspirit**
New contributor task #746

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, okay to test.

**Comment by haoyuan** [comment]
Jenkins, add to whitelist.

File: clients/unshaded/src/main/java/tachyon/client/RemoteBlockReader.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static RemoteBlockReader createRemoteBlockReader(TachyonConf conf) {
    * @param blockId the id of the block trying to read.
    * @param offset the offset of the block.
    * @param length the length the client wants to read.
-   * @return
+   * @return a byte buffer containing the remote data block.
    * @throws IOException
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/RemoteBlockReader.java]: Could you also add the conditions in which an IOException is thrown?

**Comment by stormspirit** [comment]
Added the throwing condition comment. Can you review?

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**